flow-typed/fusion-core.js

```diff
@@ -21,8 +21,12 @@ declare module 'fusion-core' {
       body: Array<string>,
     },
   } & KoaContext;
+  declare export type Token<T> = {
```

Comment:
This export is already defined [here](https://github.com/ganemone/fusion-core/blob/82b7528ce3bdc7d28b264e26fb3971d1c94bf0f5/flow-typed/fusion-core.js#L76).

Reply:
whoops good catch